config/main.py

```diff
@@ -929,6 +930,8 @@ def config(ctx):
 config.add_command(nat.nat)
 config.add_command(vlan.vlan)
 
+config.add_command(kube.kubernetes)
```

Comment:
Suggest adding this to the group above, in alphabetical order, adding a comment above and removing the NAT comment, like so:

```python
# Add groups from other modules
config.add_command(aaa.aaa)
config.add_command(aaa.tacacs)
config.add_command(feature.feature)
config.add_command(kube.kubernetes)
config.add_command(nat.nat)
config.add_command(vlan.vlan)
```

Reply:
Done. Thanks for the 'comments'.